python/txt2tags.py

```diff
@@ -1604,25 +1606,34 @@ def Readfile(file, remove_linebreaks=0, ignore_error=0):
 			if not ignore_error:
 				Error(_('You must feed me with data on STDIN!'))
 	else:
-		try: f = open(file); data = f.readlines() ; f.close()
+		try:
+			f = open(file, encoding='utf-8')
+			data = f.readlines()
+			f.close()
 		except:
 			if not ignore_error:
 				Error(_("Cannot read file:")+" %s"%file)
+		# remove UTF-8 BOM
+		if data and data[0] and data[0][0] == '\ufeff':
+			data[0] = data[0][1:]
 	if remove_linebreaks:
-		data = map(lambda x:re.sub('[\n\r]+$','',x), data)
+		data = [re.sub('[\n\r]+$','',x) for x in data]
 	Message(_("File read (%d lines): %s")%(len(data),file),2)
 	return data
 def Savefile(file, contents):
 	try: f = open(file, 'wb')
```

Comment:
Actually
```
data = [x.rstrip() for x in data]
```
is mostly the same, however, but not exactly the same.
If a line in the t2t file has trailing spaces, the current code does not remove them.
I wanted to produce exactly the same output, otherwise, it is hard to know the code has regression or not.

Reply:
I don't understand how your remark about `data = [x.rstrip() for x in data]` relates to opening the file with utf-8

Reply:
I understand you are saying about open(file, 'wb').
I remember that binary mode was necessary for producing the same outputs as with python 2 version, but I should try a better approach.

https://github.com/nvdajp/txt2tags/commit/87808f4833b631d1ff3bc396b01af5bbb858aabd

Reply:
I have looked at the code again. However, I am reluctant to change further.
SaveFile() is called with the content which is the return value of addLineBreaks().
addLineBreaks() replaces line breaks in condideration of sys.platform.
I don't want to allow platform dependent line ending handling again in SaveFile().

Reply:
Removed unnecessary string replaces.